src/main/scala/firrtl/passes/memlib/MemConf.scala

```diff
@@ -57,13 +57,13 @@ object MemConf {
   }
 
   def apply(name: String, depth: BigInt, width: Int, readPorts: Int, writePorts: Int, readWritePorts: Int, maskGranularity: Option[Int]): MemConf = {
-    val ports: Map[MemPort, Int] = (if (maskGranularity.isEmpty) {
-      (if (writePorts == 0) Map.empty[MemPort, Int] else Map(WritePort -> writePorts)) ++
-      (if (readWritePorts == 0) Map.empty[MemPort, Int] else Map(ReadWritePort -> readWritePorts))
+    val ports: Seq[(MemPort, Int)] = (if (maskGranularity.isEmpty) {
+      (if (writePorts == 0) Seq() else Seq(WritePort -> writePorts)) ++
+      (if (readWritePorts == 0) Seq() else Seq(ReadWritePort -> readWritePorts))
```

Comment:
not related too, but I think it should be `Seq` too

Reply:
Somehow Scala 2.13 inferred incompatible map types. I decided that creating a Seq of tuples and then doing toMap on it was the cleanest fix.

Reply:
yes, `Seq` here is much reasonable!